etl/steps/data/grapher/worldbank_wdi/2023-05-29/wdi.meta.yml

```diff
@@ -0,0 +1,140 @@
+dataset:
+  update_period_days: 365
+  sources: []
+definitions:
+  common:
+    sources: []
+    origins:
+      - producer: World Bank World Development Indicators
+        title: UNICEF, World Health Organization and World Bank
+        description: |-
+          The World Development Indicators (WDI) is the World Bank’s primary collection of development indicators, compiled from officially-recognized international sources. It presents the most current and accurate global development data available and includes national, regional, and global estimates.
+
+          \[Text from [World Bank data catalog](https://datacatalog.worldbank.org/dataset/world-development-indicators)\]
```

Comment:
What's the purpose of the `\[`? Is it supposed to render differently on the data page?

![image](https://github.com/owid/etl/assets/18101289/3b4ac1ff-5b4c-4339-9e84-d07844cd3ee4)